parcel/ParcelDistributions.jl

```diff
@@ -54,3 +57,27 @@ function distribution_moments(::Gamma, qₗ, Nₗ, ρₗ, ρ_air, qᵢ, Nᵢ, ρ
     end
     return (; rₗ, Aₗ, Vₗ, rᵢ)
 end
+
+function distribution_moments(::Lognormal, qₗ, Nₗ, ρₗ, ρ_air, qᵢ, Nᵢ, ρᵢ, distr_params)
+    (; σ_g) = distr_params
+    # liquid droplets
+    if Nₗ == FT(0)
+        rₗ = FT(0)
+        Aₗ = FT(0)
+        Vₗ = FT(0)
+    else
+        r_median = cbrt(3 / 4 / FT(π) / Nₗ * qₗ / ρₗ * ρ_air / exp(9 / 2 * (log(σ_g))^2))
```

Comment:
Do you have your derivation notes on how you got those formulae?

Reply:
Yes, should I add it to the documentation somewhere?

Reply:
Sure! But I just asked, because I was getting something slightly different and I wanted to double check

Reply:
LGTM! Thank you